src/lib/web-component/filter-timestamp/filter-timestamp-element.js

```diff
@@ -132,9 +130,6 @@ export class FilterTimestamp extends HTMLElement {
   handleEvent(event) {
     if (event.type === 'click') {
       switch (event.target) {
-        case this.clearFilterBtn:
-          this.onClearFilter();
-          break;
```

Comment:
Removing this since in the current UI we aren't removing both the timestamp at once. Explanation given here - https://github.com/mozilla/extension-activity-monitor/pull/51/files#r607192050